src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -102,6 +102,30 @@
                                      (@default-lang (-> field-descriptor :params :info-text :label)))]
         [markdown-paragraph info]))))
 
+(defn question-hakukohde-names [field-descriptor]
+  (let [show-hakukohde-list? (r/atom false)]
+    (fn [field-descriptor]
+      (let [lang                           @(subscribe [:application/form-language])
+            tarjonta-hakukohtet            @(subscribe [:application/tarjonta-hakukohteet])
+            selected-hakukohteet           @(subscribe [:application/selected-hakukohteet])
+            field-hakukohteet              (:belongs-to-hakukohteet field-descriptor)
+            selected-hakukohteet-for-field (clojure.set/intersection (set field-hakukohteet)
+                                                                     (set selected-hakukohteet))
+            selected-hakukohde-names       (->> tarjonta-hakukohtet
+                                                (filter #(some #{(:oid %)} selected-hakukohteet-for-field))
+                                                (map :name)
+                                                (map #(some % [lang :fi :sv :en])))]
+        [:div.application__question_hakukohde_names_container
+         [:a
+          {:on-click #(reset! show-hakukohde-list? (not @show-hakukohde-list?))}
```

Comment:
Vähän tiiviimpi ois `(swap! show-hakukohde-list? not)`

Reply:
Se ei toiminu jostain syystä kun koitin sitä aluksi. Koitan vielä..